src/lib.rs

```diff
@@ -1056,17 +1159,14 @@ impl<T: Copy> Unalign<T> {
     }
 }
 
-// SAFETY: Since `T: AsBytes`, we know that it's safe to construct a `&[u8]`
-// from an aligned `&T`. Since `&[u8]` itself has no alignment requirements, it
-// must also be safe to construct a `&[u8]` from a `&T` at any address. Since
-// `Unalign<T>` is `#[repr(C, packed)]`, everything about its layout except for
-// its alignment is the same as `T`'s layout.
-unsafe impl<T: AsBytes> AsBytes for Unalign<T> {
-    fn only_derive_is_allowed_to_implement_this_trait()
-    where
-        Self: Sized,
-    {
-    }
+safety_comment! {
+    /// SAFETY:
+    /// Since `T: AsBytes`, we know that it's safe to construct a `&[u8]` from
+    /// an aligned `&T`. Since `&[u8]` itself has no alignment requirements, it
+    /// must also be safe to construct a `&[u8]` from a `&T` at any address.
+    /// Since `Unalign<T>` is `#[repr(C, packed)]`, everything about its layout
+    /// except for its alignment is the same as `T`'s layout.
+    unsafe_impl!(T: AsBytes => AsBytes for Unalign<T>);
```

Comment:
Also `FromBytes`?

Reply:
jk, we already implement it:
https://github.com/google/zerocopy/blob/140b89c7ad5989459b550f3f198c2ef8a7bffcf3/src/lib.rs#L887-L889